plugins/Toolbox/src/Toolbox.py

```diff
@@ -576,52 +556,85 @@ def isLoadingComplete(self) -> bool:
     # Make API Calls
     # --------------------------------------------------------------------------
     def _makeRequestByType(self, request_type: str) -> None:
-        Logger.log("d", "Requesting '%s' metadata from server.", request_type)
-        request = QNetworkRequest(self._request_urls[request_type])
-        for header_name, header_value in self._request_headers:
-            request.setRawHeader(header_name, header_value)
+        Logger.log("d", "Requesting [%s] metadata from server.", request_type)
         self._updateRequestHeader()
-        if self._network_manager:
```

Comment:
You are refactoring the `_makeRequestByType` and removing the `_network_manager` here (therefore removing the `_onRequestFinished` slot), replacing it with a callback `_onAuthorsDataRequestFinished`, which is pretty much the same function as the `_onRequestFinished`, BUT is missing few parts, one of which the check if the request_type is "subscribed_packages" (something I've been working on for the last sprint).
Is this left out intentionally or am I missing something?

https://github.com/Ultimaker/Cura/pull/6845/files#diff-a2d4773eeefc7f207c5a6a0d637a2c4fR680
Here's what I'm talking about.

I realize that you might have wanted to use this only for Authors (i.e. Materials), hence naming it `_onAuthorsDataRequestFinished`, but in fact this same callback is called on any call, including those for packages and updates.

Reply:
Also a good point. Confused me as well